vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/transformer/SiteMapTableTransformer.java

```diff
@@ -58,10 +58,17 @@
                 Set<String> siteMapRelativeUrls;
                 try
                 {
-                    siteMapRelativeUrls = filterResults(siteMapParser.parse(true, mainApplicationPage,
+                    if (mainApplicationPage != null)
```

Comment:
also I believe the fix should be implemented in `getMainApplicationPageUri()` to avoid the same issue in `HeadlessCrawlerTableTransformer`